base_user_role/models/role.py

```diff
@@ -23,6 +23,10 @@ class ResUsersRole(models.Model):
         'res.users.role.line', 'role_id', string=u"Users")
     user_ids = fields.One2many(
         'res.users', string=u"Users", compute='_compute_user_ids')
+    group_category_id = fields.Many2one(
+        related='group_id.category_id',
+        default=lambda cls: cls.env.ref(
+            'base_user_role.ir_module_category_role').id)
```

Comment:
It does not work without the `related` part. About the field's name, I don't know what is best.